install/receivers.csv

```diff
@@ -177,11 +177,13 @@ Trimble Navigation Ltd.,TRIMBLE ALLOY,6136R40009,NLS4,2022-08-24T01:22:38Z,9999-
 Trimble Navigation Ltd.,TRIMBLE ALLOY,6136R40064,CLIM,2022-08-30T23:01:38Z,9999-01-01T00:00:00Z
 Trimble Navigation Ltd.,TRIMBLE ALLOY,6139R40184,WMAT,2022-08-09T23:40:08Z,9999-01-01T00:00:00Z
 Trimble Navigation Ltd.,TRIMBLE ALLOY,6139R40213,VGET,2023-01-16T01:46:20Z,9999-01-01T00:00:00Z
```

Comment:
I think this previous line was not deleted, this might be causing the test to fail?